website/docs/configuration/expressions.html.md

```diff
@@ -272,18 +276,28 @@ for use in references, as follows:
 
   To obtain a map of values of a particular argument for _labelled_ nested
   block types, use a [`for` expression](#for-expressions):
-  `[for k, device in aws_instance.example.device : k => device.size]`.
+  `{for k, device in aws_instance.example.device : k => device.size}`.
```

Comment:
Note, this was something I fixed in passing, because next to its related text this looks correct. Please speak up if it doesn't look right!

Reply:
Using `{` and `}` instead of `[` and `]` is correct.

Reply:
I appreciate the confirmation Roger!